pipeline/processors/metrics-selector.md

```diff
@@ -1,27 +1,43 @@
 # Metrics Selector
 
-The **metric_selector** processor allows you to select metrics to include or exclude (similar to the `grep` filter for logs).
+The _Metrics Selector_ processor lets you choose which metrics to include or exclude, similar to the [Grep](../pipeline/filters/grep) filter for logs.
 
-<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=326269f3-cfea-472d-9169-1de32c142b90" />
```

Comment:
I don't think this should be deleted - it's a tracking pixel. all of the pages have it. or at least a lot of them do